packages/react-components/react-tag-picker/stories/src/TagPicker/TagPickerAppearance.stories.tsx

```diff
@@ -28,11 +29,18 @@ const Example = ({ appearance }: Pick<TagPickerProps, 'appearance'>) => {
   };
   const tagPickerOptions = options.filter(option => !selectedOptions.includes(option));
 
+  const labelColor =
+    appearance === 'filled-lighter' || appearance === 'filled-darker'
+      ? tokens.colorNeutralForegroundInverted2
+      : tokens.colorNeutralForeground1;
+
+  const label = <Label style={{ color: labelColor }}>Select Employees</Label>;
+
   return (
-    <Field label="Select Employees" style={{ maxWidth: 400 }}>
+    <Field label={label} style={{ maxWidth: 400 }}>
       <TagPicker appearance={appearance} onOptionSelect={onOptionSelect} selectedOptions={selectedOptions}>
         <TagPickerControl>
-          <TagPickerGroup>
+          <TagPickerGroup aria-label="Select Employees">
```

Comment:
Shouldn't we perhaps consider getting the label from the `Field` context? 🤔

Reply:
We don't really want the listbox label to match the input label, so I don't think we can get both from Field.

related 😁:
```suggestion
          <TagPickerGroup aria-label="Selected Employees">
```